src/geo/map/torque-layer.js

```diff
@@ -30,6 +33,20 @@ var TorqueLayer = LayerModelBase.extend({
     time: undefined // should be a Date instance
   },
 
+  // Helper method to be used from a few places, it parses torque cartocss to get
+  // the animation duration or a default duration
+  getAnimationDuration: function (cartocss) {
```

Comment:
Instead of using RegExp, let's keep using a CSS parser (like in turbo-carto, deep-insights).